tests/baselines/reference/recursiveTypeRelations.errors.txt

```diff
@@ -0,0 +1,48 @@
+tests/cases/compiler/recursiveTypeRelations.ts(8,5): error TS2391: Function implementation is missing or not immediately following the declaration.

```

Comment:
It's a repro and the repro has syntax errors (but previously would never compile because of the stack overflow).